bats/tests/helpers/defaults.bash

```diff
@@ -188,3 +193,47 @@ validate_enum RD_LOCATION system user dist dev ""
 using_dev_mode() {
     [ "$RD_LOCATION" = "dev" ]
 }
+
+########################################################################
+# RD_K3S_VERSIONS specifies a list of k3s versions. foreach_k3s_version()
+# can dynamically register a test to run once for each version in the
+# list. Only versions between RD_K3S_MIN and RD_K3S_MAX (inclusively)
+# will be used.
+#
+# Special values:
+# "all" will fetch the list of all k3s releases from GitHub
+# "latest" will fetch the list of latest versions from the release channel
+
+: "${RD_K3S_MIN:=1.0.0}"
+: "${RD_K3S_MAX:=1.99.0}"
+: "${RD_K3S_VERSIONS:=}"
```

Comment:
This defaults to empty, so no tests are run, right? It might be good to call that out.

Reply:
I only did this because running `tests/k8s/foreach-k3s-version` with just the default Kubernetes version is redundant with other tests. But the additional time when you run the full test is minimal, so I've now added

```
: "${RD_K3S_VERSIONS:=RD_KUBERNETES_PREV_VERSION}"
```